tests/publiccloud/slem_basic.pm

```diff
@@ -25,13 +25,42 @@ sub has_wicked {
     return 0;
 }
 
+sub check_avc {
+    my ($self) = @_;
+
+    my $instance = $self->{my_instance};
+    # Read the Access Vector Cache to check for SELinux denials
+    my $avc = $instance->ssh_script_output(cmd => 'sudo ausearch -ts boot -m avc --format raw | grep type=AVC');
+    record_info("AVC at boot", $avc);
+    return if ($avc =~ "no matches");
+
+    ## Gain better formatted logs and upload them for further investigation
+    $instance->ssh_assert_script_run(cmd => 'sudo ausearch -ts boot -m avc > ausearch.txt');
+    assert_script_run("scp " . $instance->username() . "@" . $instance->public_ip . ":ausearch.txt ausearch.txt");
+    upload_logs("ausearch.txt");
+
+    # TODO: Uncomment once all ongoing issues are resolved. For now there will be only a softfailure
+    #die "SELinux access denials on first boot";
+
+    # Record a soft_failure for each line of the AVC
+    my @avc = split(/\n/, $avc);
+    for my $row (@avc) {
+        $row =~ s/^\s+|\s+$//g;
+        record_soft_failure($row) unless ($row eq '');
```

Comment:
is there some bug ?

Reply:
Please read the comment in line 42 again.

Reply:
I now changed it to a `record_info`, which work better here.